core/statistics/tpsBenchmark.go

```diff
@@ -36,9 +50,47 @@ type ShardStatistics struct {
 	totalProcessedTxCount *big.Int
 }
 
+// NewTPSBenchmarkWithInitialData instantiates a new object responsible with calculating statistics for each shard tps
+// starting with initial data
+func NewTPSBenchmarkWithInitialData(
+	appStatusHandler core.AppStatusHandler,
+	initialTpsBenchmark *TpsPersistentData,
+	nrOfShards uint32,
+	roundDuration uint64) (*TpsBenchmark, error) {
```

Comment:
indenting. add "," after roundDuration uint64 and move the rest to next line

Reply:
done